packages/editor/src/components/document-outline/index.js

```diff
@@ -95,7 +87,11 @@ export const DocumentOutline = ( { blocks = [], title, onSelect, isTitleSupporte
 					<DocumentOutlineItem
 						level={ __( 'Title' ) }
 						isValid
-						onClick={ focusTitle }
+						onSelect={ ( e ) => {
+							removeURLHash();
+							onSelect( e );
```

Comment:
Is there a specific reason we pass through the event? I don't know that we need to or want to make this interface commitment.

Reply:
nope, reflexive. removed in https://github.com/WordPress/gutenberg/pull/10815/commits/613bc47bb2a98ce306ee43e00f0753160963f1d2